tools/memento-cid-differential/generate_golden.py

```
@@ -0,0 +1,46 @@
+CORPUS_DIR = HERE / "corpus"
+OUT_PATH = HERE / "golden_mementos.jsonl"
+
+
+def main() -> int:
+    files = sorted(CORPUS_DIR.glob("*.py"))
+    if not files:
+        print("no corpus files found", file=sys.stderr)
+        return 1
+    lines = []
+    for f in files:
+        result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)